agendas.md

```diff
@@ -10,6 +10,7 @@ title: Agendas
 ## 2023
 
 ### May
+- [Committee Handover Meeting - Monday 29th May] (/assets/agendas/2023/05/29.pdf)
```

Comment:
the space between the closing square bracket "]" and the opening bracket "(" needs to be removed

Reply:
done